### PR TITLE
added tests for auto-selecting delimiter

### PR DESCRIPTION
**Pull request by gregllong**
added two fixture files for testing, too

### CODE REVIEW

**Comment by jayich** [comment]
@gregllong , see this file here for CSV:

https://github.com/Spinmob/spinmob/blob/master/tests/fixtures/data_types/CSV_xy.dat

This is the first semicolon test, which was lacking.

**Comment by gregllong** [comment]
@jayich That file is not what I was interested in in the other pull request. I have just added another file with ";" as the delimiter, and all tests passed.

File: tests/fixtures/data_types/semicolumn.dat [hunk 1]
```diff
@@ -0,0 +1,34 @@
+85; .158
```
Comment (jayich) [inline tests/fixtures/data_types/semicolumn.dat]: final name edit: semicolon.dat

File: tests/test__data.py [hunk 2]
```diff
@@ -44,6 +44,9 @@ def setUp(self):
 
         self.databoxCSV = _dt.databox(delimiter=', ')
 
+        filename = 'semicolumn.dat'
```
Comment (jayich) [inline tests/test__data.py]: file name edit.

File: tests/test__data.py [hunk 3]
```diff
@@ -44,6 +44,9 @@ def setUp(self):
...
+        self.data_path_semicolumn = os.path.join(midPath, filename)
```
Comment (jayich) [inline tests/test__data.py]: semicolon here too.

**Comment by jayich** [comment]
@gregllong , I think you still need to change the fixture name.

**Comment by jayich** [comment]
LGTM.  @gregllong , if you are not planning on making anymore changes I can merge this in.

**Comment by gregllong** [comment]
@jayich That sounds good.

**Pull request merged**